client/sections.js

```diff
@@ -22,21 +22,20 @@ const extensionSections = extensions.map( extension => {
 	}
 } );
 
-if ( config.isEnabled( 'devdocs' ) ) {
-	sections.push( {
-		name: 'devdocs',
-		paths: [ '/devdocs' ],
-		module: 'devdocs',
-		secondary: true,
-		enableLoggedOut: true
-	} );
+sections.push( {
```

Comment:
I think I found solution with https://github.com/Automattic/wp-calypso/pull/11352/commits/3d98ba8700a7764e54ded8fc9cfe3d0f3be14341 to this issue. This allows to bundle all sections and extensions, but at the same time never load chunks that are hidden behind the feature flag using `env_id` in `package.json` file.

@mtias @mcsf @ehg Let us know if that solution would work :)